commands/utils/validate.js

```diff
@@ -227,14 +238,20 @@ module.exports = validate_config = function (lt_config, validation_configs) {
             reject(
               "Error!! Reporter JSON File has no keys, either remove Key reporter_config_file from lambdatest config or pass some options"
             );
+          }else if (reporter_config.reporterEnabled && reporter_config.reporterEnabled != ""){
+            if (!reporter_config.reporterEnabled.includes("mochawesome")){
+              reject("Error!! Please add mochawesome in reporterEnabled and add the related config");
```

Comment:
Can we give a warning message instead of directly rejecting the test

Reply:
makes sense